src/SongPosition.h

```diff
@@ -6,41 +6,54 @@
 #include "LightsManager.h"
 #include "MessageManager.h"
 #include "TimingData.h"
-//XXX: where does this come from? might need another include
 struct lua_State;
 
 class SongPosition
 {
 public:
-	// Arcade - the current stage (one song).
```

Comment:
Let's keep the comments. I think they're helpful here.

Reply:
> Let's keep the comments. I think they're helpful here.

Done